META.sh

```diff
@@ -1,5 +1,7 @@
 #!/bin/bash
-# SPDX-License-Identifier: Apache-2.0
+# Copyright (c) The mlkem-native project authors
+# Copyright (c) The mldsa-native project authors
+# SPDX-License-Identifier: CC-BY-4.0
```

Comment:
Can we change this to `Apache-2.0 OR ISC OR MIT`

Reply:
Done